src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/StringImport.cs

```diff
@@ -33,5 +31,11 @@ protected override string GetName(NodeFactory context)
         {
             return "StringCell: " + _token.ToString();
         }
+
+        // This is just here in case of future extension (everything is already compared in the base CompareToImpl)
```

Comment:
This is suspicious to me... I think we need to compare _token here before calling the base method

Reply:
The token is compared in the base class (as part of StringImportSignature).

Reply:
Ok I see that. We get the comparison correctly because the base ends up comparing the StringImportSignature.
Can you add a comment explainig this?

Reply:
Yep, will do